src/bigquery/bigquery.py

```diff
@@ -0,0 +1,142 @@
+# Copyright © 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Dharmendra G Patel <dhpatel@redhat.com>
+#
+"""Bigquery implementation to read big data for manifest files."""
+import time
+from rudra import logger
+from rudra.data_store.aws import AmazonS3
+from src.config.settings import AWS_SETTINGS
+from src.bigquery.base import BigqueryBuilder, DataProcessing
+from src.bigquery.base_collector import BaseCollector
+from src.bigquery.maven_collector import MavenCollector
+from src.bigquery.npm_collector import NpmCollector
+from src.bigquery.pypi_collector import PypiCollector
+
+ECOSYSTEM_MANIFEST_MAP = {
+    'maven': 'pom.xml',
+    'npm': 'package.json',
+    'pypi': 'requirements.txt',
+}
+
+
+class Bigquery(BigqueryBuilder):
```

Comment:
I dont see any need for this class. The base class can have query method which can take in a config and a query. That way it becomes generic to execute any query.

Reply:
I will remove this base class, we don't need that anymore as we have only one query to execute. This was done before as we were running this queries separate for each ecosystem.